docs/reference/components.cue

```diff
@@ -487,14 +492,408 @@ components: {
-					type: "gauge"
+					type:              "gauge"
+					default_namespace: "vector"
 				}
 			}
 		}
 
 		telemetry: metrics: {
 			// Default metrics for each component
-			vector_events_processed_total: _vector_events_processed_total
+			events_processed_total: _events_processed_total
+			processed_bytes_total:  _processed_bytes_total
+
+			// Reusable metric definitions
+			_auto_concurrency_averaged_rtt: {
+				description:       "The average round-trip time (RTT) from the HTTP sink across the current window."
+				type:              "histogram"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_auto_concurrency_in_flight: {
+				description:       "The number of outbound requests from the HTTP sink currently awaiting a response."
+				type:              "histogram"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_auto_concurrency_limit: {
+				description:       "The concurrency limit that the auto-concurrency feature has decided on for this current window."
+				type:              "histogram"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_auto_concurrency_observed_rtt: {
+				description:       "The observed round-trip time (RTT) for requests from this HTTP sink."
+				type:              "histogram"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_checkpoint_write_errors_total: {
+				description:       "The total number of errors writing checkpoints."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_checkpoints_total: {
+				description:       "The total number of files checkpointed."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_checksum_errors: {
+				description:       "The total number of errors identifying files via checksum."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags & {
+					file: _file
+				}
+			}
+			_events_discarded_total: {
+				description:       "The total number of events discarded by this component."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_events_processed_total: {
+				description:       "The total number of events processed by this component."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _component_tags & {
+					file: _file
+				}
+			}
+			_file_delete_errors: {
+				description:       "The total number of failures to delete a file."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags & {
+					file: _file
+				}
+			}
+			_file_watch_errors: {
+				description:       "The total number of errors caused by failure to watch a file."
```

Comment:
This sounds like "the total number of errors caused by us failing to watch a file". Maybe something like "Times erred to watch a file.", or something in line with the other descriptions like "The total number of failures to watch a file."